opendebates/templates/base.html

```diff
@@ -187,6 +188,37 @@
 
 <script type="text/javascript">
   $(window).load(function() {
+    var versions = [
+      ["Firefox", $.browser.mozilla, 55],
```

Comment:
We're supposed to warn for Firefox below Firefox 44, I think.

Oh actually there's another Firefox at line 195, so this one can probably just be dropped.